src/Client/Stripe.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace MatchBot\Client;
+
+use MatchBot\Domain\Currency;
+use MatchBot\Domain\Money;
+use MatchBot\Domain\StripeCustomerId;
+use Stripe\StripeClient;
+
+class Stripe
```

Comment:
I wanted to make a class just to isolate how we talk to stripe. The off the shelf stripe client is very tedious to double with prophecy, and overly complicated to use. In principle we should also have a unit test for this class, but as the implementation of fetchBalance is largely copied from Identity I'm quite confident that it's correct.

Reply:
I think this is reasonable, but I'd also be keen for us to add a test in the next couple of weeks while we're in "code slush", since this is newly becoming our abstraction and is a particularly important area.